packages/selection/src/useSelection.js

```diff
@@ -161,6 +162,16 @@ export function useSelection({
     }
   }, [controlledFocusedItem, items, refs]);
 
+  useEffect(() => {
+    if (selectedItem === undefined && defaultSelectedIndex !== undefined) {
+      dispatch({
+        type: ACTIONS.KEYBOARD_SELECT,
+        payload: items[defaultSelectedIndex],
+        items
+      });
+    }
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Reason for this is I do not care if any of this effects dependencies change and only want it to run on first mount.